Doc/c-api/sys.rst

```diff
@@ -78,9 +78,9 @@ Operating System Utilities
 
    Return true when the interpreter runs out of stack space.  This is a reliable
    check, but is only available when :const:`USE_STACKCHECK` is defined (currently
-   on Windows using the Microsoft Visual C++ compiler).  :const:`USE_STACKCHECK`
-   will be defined automatically; you should never change the definition in your
-   own code.
+   on macOS and Windows using the Microsoft Visual C++ compiler).
```

Comment:
I have to check with a native speaker, but this seems to suggest that USE_STACKCHECK is only enabled on macOS when using the Microsoft compiler, which is not what you're trying to say here. 

That said, I'm not a native english speaker and might interpret this sentence too strictly.